persistence/jdbc/src/test/java/org/infinispan/persistence/jdbc/stringbased/StringStoreWithPooledConnectionFunctionalTest.java

```diff
@@ -0,0 +1,36 @@
+package org.infinispan.persistence.jdbc.stringbased;
+
+import org.infinispan.persistence.jdbc.configuration.ConnectionFactoryConfiguration;
+import org.infinispan.persistence.jdbc.configuration.JdbcStringBasedStoreConfigurationBuilder;
+import org.infinispan.persistence.jdbc.configuration.PooledConnectionFactoryConfiguration;
+import org.infinispan.persistence.jdbc.connectionfactory.ConnectionFactory;
+import org.infinispan.persistence.jdbc.impl.connectionfactory.PooledConnectionFactory;
+import org.testng.annotations.Test;
+
+import static org.infinispan.test.fwk.UnitTestDatabaseManager.configureUniqueConnectionFactory;
+
+@Test(groups = "functional", testName = "persistence.jdbc.stringbased.StringStoreWithPooledConnectionFunctionalTest")
+public class StringStoreWithPooledConnectionFunctionalTest extends AbstractStringBasedCacheStore {
+    private ConnectionFactoryConfiguration customFactoryConfiguration;
+
+    public StringStoreWithPooledConnectionFunctionalTest() {}
+
+    //Invoked from external resource
+    public StringStoreWithPooledConnectionFunctionalTest(PooledConnectionFactoryConfiguration customFactoryConfiguration) {
```

Comment:
We should add the annotation `@SuppressWarnings("unused")` so that a warning is not displayed in the IDE